rpm/0005-Support-Microsoft-OAuth2-flow.patch

```diff
@@ -142,5 +145,5 @@ index 3eb3fef..dc1dd7e 100644
                                    GrantType::AuthorizationCode);
          }
 -- 
-2.37.2
+2.37.3
 
```

Comment:
Please use `git format-patch --no-numbered --zero-commit --no-signature --full-index` to generate the patch.

Reply:
It might also make sense to upstream the patch.

Reply:
> Please use `git format-patch --no-numbered --zero-commit --no-signature --full-index` to generate the patch.

There you go. :smiley: